src/org/mozilla/javascript/typedarrays/Conversions.java

```diff
@@ -14,33 +14,14 @@
 
 public class Conversions
 {
-    public static final int EIGHT_BIT = 1 << 8;
-    public static final int SIXTEEN_BIT = 1 << 16;
-    public static final long THIRTYTWO_BIT = 1L << 32L;
-
     public static int toInt8(Object arg)
     {
-        int iv;
-        if (arg instanceof Integer) {
-            iv = (Integer)arg;
```

Comment:
Note that this fast-path was completely redundant, because [`ScriptRuntime.toInt32`](https://github.com/mozilla/rhino/blob/f192525fa96ad2c1211ec984e300e475bf7e4bb6/src/org/mozilla/javascript/ScriptRuntime.java#L1236) already takes care of this optimization.